example/0-setup.sh

```diff
@@ -35,6 +36,7 @@ for dir in $DATA_DIRS; do
     ln -svfT "$PERSISTENT_TMP_DIR/$dir" "$BIN_DIR/$dir"
 done
 
+<<<<<<< HEAD
```

Comment:
looks like you have multiple merge conflicts in the commit
search for `<<<<<<<` all around

Reply:
Sorry, fixed that